### PR TITLE
Add line recognition

### PR DESCRIPTION
**Pull request by steffi3011**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/SixLabors/Fonts/pulls) open
- [x] I have verified that I am following matches the existing coding patterns and practice as demonstrated in the repository. These follow strict Stylecop rules :cop:.
- [x] I have provided test coverage for my change (where applicable)

### Description

Add line recognition to determine the number of rows needed in the generated layout.

### CODE REVIEW

File: src/SixLabors.Fonts/TextMeasurer.cs [hunk 1]
```diff
@@ -58,6 +58,24 @@ public static FontRectangle MeasureBounds(ReadOnlySpan<char> text, TextOptions o
...
+        /// <returns>The count of lines used.</returns>
+        public static int CountLines(string text, TextOptions options)
+            => TextMeasurerInt.Default.CountLines(text.AsSpan(), options);
+
+        /// <summary>
+        /// Counts the lines from the generated layout.
+        /// </summary>
+        /// <param name="text">The text.</param>
+        /// <param name="options">The style.</param>
+        /// <returns>The count of lines used.</returns>
```
Comment (JimBobSquarePants) [inline src/SixLabors.Fonts/TextMeasurer.cs:76]:
```suggestion
        /// <summary>
        /// Gets the number of lines contained within the text.
        /// </summary>
        /// <param name="text">The text.</param>
        /// <param name="options">The style.</param>
        /// <returns>The line count.</returns>
```

File: src/SixLabors.Fonts/TextMeasurer.cs [hunk 2]
```diff
@@ -58,6 +58,24 @@ public static FontRectangle MeasureBounds(ReadOnlySpan<char> text, TextOptions o
         /// <returns>Whether any of the characters had non-empty bounds.</returns>
         public static bool TryMeasureCharacterBounds(ReadOnlySpan<char> text, TextOptions options, out GlyphBounds[] characterBounds)
             => TextMeasurerInt.Default.TryMeasureCharacterBounds(text, options, out characterBounds);
+        
+        /// <summary>
+        /// Counts the lines from the generated layout.
+        /// </summary>
+        /// <param name="text">The text.</param>
+        /// <param name="options">The style.</param>
+        /// <returns>The count of lines used.</returns>
```
Comment (JimBobSquarePants) [inline src/SixLabors.Fonts/TextMeasurer.cs:67]:
```suggestion
```suggestion
        /// <summary>
        /// Gets the number of lines contained within the text.
        /// </summary>
        /// <param name="text">The text.</param>
        /// <param name="options">The style.</param>
        /// <returns>The line count.</returns>
```
```

**Review by JimBobSquarePants — CHANGES_REQUESTED** [review]
Thanks for this submission @steffi3011 I really appreciate the assistance. 

It looks like we're missing unit tests plus there's some StyleCop issues. I've also provided suggestion for the intellisense documentation. 

Please let me know if you need help with anything.

**Review by JimBobSquarePants — APPROVED** [review]
Fantastic, thanks for your help 👍

**Comment by steffi3011** [comment]
You are welcome.
Could you tell me when a new version with this feature will be released?

**Comment by JimBobSquarePants** [comment]
It’s available in our MyGet feed now. We don’t have a timeframe for the next NuGet release

**Comment by steffi3011** [comment]
For us, this new version with this special feature would be very important. Maybe it is possible to make the NuGet release as soon as possible?

**Comment by JimBobSquarePants** [comment]
@steffi3011 Does your company have a [SixLabors License](https://sixlabors.com/pricing/)?

**Pull request merged**